docs/guide/en/built-in-rules-compare.md

```diff
@@ -0,0 +1,40 @@
+# `Compare` - comparing validated value with target value
+
+## Using with `DateTime` objects
```

Comment:
Is it necessary if it now works without setting the type?

Reply:
I think it still can be useful.